index1.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
change the lang to pt-br  @GenniferSebastiao

Reply:
Boas Gennifer, parabéns pelo código, após a analise está resumido e bem feito. Porém sugiro a nomeclatura das classes dos botões relaciona-los as suas acções porque se estiveres a codar uma app muito grande poderás encontrar dificuldades na altura de resolver um bug com um determinado botão. O teu código está limpo simples e bem explicito.

Reply:
Boas Gennifer, parabéns pelo código, após a analise está resumido e bem feito. Porém sugiro a nomeclatura das classes dos botões relaciona-los as suas acções porque se estiveres a codar uma app muito grande poderás encontrar dificuldades na altura de resolver um bug com um determinado botão. O teu código está limpo simples e bem explicito.